tests/.eslintrc.js

```diff
@@ -1,5 +1,13 @@
 module.exports = {
   env: {
     'embertest': true
+  },
+  globals: {
+    'Factory': false,
+    'assertGithubUserOk':false,
+    'assertGithubRepositoryOk': false,
+    'assertGithubBranchOk': false,
+    'assertGithubReleaseOk': false,
+    'assertGithubOrganizationOk': false
```

Comment:
These asserts should be added to `QUnit.assert` rather than treated as globals. In the helper files, we can just do `QUnit.assert.githubUserOk = function () {`, and then in the tests themselves do `assert.githubUserOk()` instead of `assertGithubUserOk()`.